content/en/seachat/seachat-manual/03-add-knowledge/04-template-file-upload.md

```diff
@@ -80,6 +80,15 @@ SeaChat offers CSV and JSON template files that you can download and fill in wit
 }]
 ```
 
+1. **id**: Unique identifier for the document.
+2. **title**: Title of the document.
+3. **text**: Content of the document that the AI agent will interpret.
+4. **custom**: Additional information that the AI agent can refer to. For example, when you need to refine a knowledge base response, you can add a custom key-value pair to the document for the AI agent to refer to.
+4. **references**: Additional information or links that the AI agent can provide. Where **title** is the name of the reference that will render in the UI and **url** is the link.
```

Comment:
i asked to show a picture side by side with the json/csv templates so people know which is which.